projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -938,37 +939,37 @@
   "item.ars_nouveau.dowsing_rod": "占卜杖",
   "effect.ars_nouveau.magic_find": "探测魔法",
   "tooltip.essences": "在灌注室创造的魔法精华。",
-  "ars_nouveau.page.dowsing_rod": "一种占卜杖，让使用者能短暂地探知紫水晶，并获得探测魔法状态。该状态让使用者75格内的魔法生物发光。单个占卜杖的使用次数有限。",
-  "ars_nouveau.apparatus.norecipe": "找不到配方。基座必须放置在3格内。",
-  "ars_nouveau.imbuement.norecipe": "找找不到配方。合成需要的基座必须放置在灌注室1格范围内。",
+  "ars_nouveau.page.dowsing_rod": "使用后能短暂地探知紫水晶，并使得让使用者75格范围内的魔法生物发光。单个占卜杖的使用次数有限。",
+  "ars_nouveau.apparatus.norecipe": "找不到配方。基座必须被放置在3格内。",
+  "ars_nouveau.imbuement.norecipe": "找不到配方。合成需要的基座必须放置在灌注室1格范围内。",
   "ars_nouveau.spell_book_gui.select": "选择",
   "ars_nouveau.levels_required": "需要等级：%s",
   "ars_nouveau.crafting": "合成：%s",
   "ars_nouveau.all_glyphs": "全部魔符",
   "ars_nouveau.not_enough_exp": "合成魔符所需经验不足。",
   "ars_nouveau.exp": "需要经验等级：%s",
   "ars_nouveau.scribing": "抄写法术",
-  "ars_nouveau.home_set": "已设定家园。",
+  "ars_nouveau.home_set": "已设置家园。",
   "ars_nouveau.gathering_at": "聚集于：%s",
   "entity.ars_nouveau.amethyst_golem": "紫晶傀儡",
   "tooltip.ars_nouveau.amethyst_charm": "通过在紫水晶母岩旁施展启蒙仪式获得",
   "item.ars_nouveau.amethyst_golem_charm": "紫晶傀儡护符",
   "ars_nouveau.page1.amethyst_golem_charm": "紫晶傀儡会种植并收获家园附近的紫水晶。在紫晶母岩旁施展启蒙仪式可以制造紫晶傀儡，其死亡后会掉落紫晶护符。",
   "ars_nouveau.page2.amethyst_golem_charm": "",
-  "ars_nouveau.page3.amethyst_golem_charm": "对方块使用紫晶护符可以召唤紫晶傀儡。在让紫晶傀儡进行紫晶种植前，必须先设定它们的家园。手持支配之杖，先右击傀儡再右击方块，紫晶傀儡就会将方块10格内视为家园。",
+  "ars_nouveau.page3.amethyst_golem_charm": "对方块使用紫晶护符可以召唤紫晶傀儡。手持支配之杖，先右击傀儡再右击方块，紫晶傀儡就会将此地视为家园，并在家园的10格范围内工作。只有设置了家园的紫晶傀儡才会正常工作。",

```

按原意是将10个范围内视为家园，这个修改改变了原意
```suggestion
  "ars_nouveau.page3.amethyst_golem_charm": "对方块使用紫晶护符可以召唤紫晶傀儡。只有设定了家园的紫晶傀儡才能进行紫晶种植。手持支配之杖，先右击傀儡再右击方块，紫晶傀儡就会将此方块周围10格内视为家园，并在此范围内工作。",
```

---

经过测试，只有“设定的‘家园’地点是容器”时才能激活拾取功能，如果这里仍然使用原意的话，会和下文关于“拾取掉落的紫水晶碎片”的行为描述不符。

---

*原文应该指的是“设定家的地点”并且“视附近范围为‘家园’”，工作时在“家园”内寻找工作目标，存放时寻找“家”的位置的容器
用了同一个词，但是实际含义不同

---

我指的是紫晶傀儡是`将该方块视为家园，在周围10格工作`还是`将该方块周围10格视为家园，并在此工作`的区别，和该方块是不是容器没有关系

---

因为后文描述是“家园在容器上”，并且实际测试表现是“只有设定的方块就是容器时才能工作”，所以此处应该是“将该方块视为家园，在周围10格工作”